tests/auth/auth.gui.ts

```diff
@@ -0,0 +1,71 @@
+import { test } from "@guardianui/test";
+import { expect } from "@playwright/test";
+
+import { common, getBaseURL } from "../utils";
+
+test.use({
+  baseURL: getBaseURL(),
+});
+
+test.describe("Auth", () => {
+  test.beforeEach(async ({ page, gui }) => {
+    const POLYGON = 137;
+    await gui.initializeChain(POLYGON, 45735806);
+    await page.goto("/");
+
+    await gui.setBalance(common.gloAddress, "12340000000000000000000");
+
+    await page.getByTestId("tos-checkbox").check();
+  });
+
+  // test("should allow email sign up", async ({ page, gui }) => {
```

Comment:
alsways good idea is too test.disable instead commenting

Reply:
using `test.skip`